Resources/SteamLanguage/enums.steamd

```diff
@@ -1205,3 +1205,36 @@ enum EUCMFilePrivacyState flags
 
 	All = Public | FriendsOnly | Private; // 14
 };
+
+enum ELobbyType
+{
+    Private = 0;
+    FriendsOnly = 1;
+    Public = 2;
+    Invisible = 3;
+};
+
+enum ELobbyFilterType

```

Comment:
This is not a Steam enum, is it?

Reply:
@xPaw, it's part of the Steam API, yes. It's just not publicly documented by Valve because it's only used internally.

Reply:
Where did you find the name though?

Reply:
Well, it's reverse engineered, so _I've_ provided a semantic name to what would otherwise be a magic constant.